python_modules/libraries/dagster-dbt/dagster_dbt/cli/resources_v2.py

```diff
@@ -254,6 +254,9 @@ def stream_raw_events(self) -> Iterator[DbtCliEventMessage]:
             log: str = raw_line.decode().strip()
             event = DbtCliEventMessage.from_log(log=log)
 
+            # Re-emit the logs from dbt CLI process into stdout.
+            print(str(event))  # noqa: T201
```

Comment:
nit: `sys.stdout.write()` would save us from having to have a noqa here.

Reply:
🙏